### ISSUE TITLE
Missing file CodelistDictionary-v32.xsl on http://standards.iso.org

### ISSUE THREAD

**Issue by heidivanparys** [ISO-TC211/XML#181]
File CodelistDictionary-v32.xsl is missing in http://standards.iso.org/iso/19115/resources/Codelist/gml

When opening an XML-file in Firefox, this results in the following error:

![image](https://user-images.githubusercontent.com/11427611/34779068-6ddbf9ae-f61f-11e7-9bae-69b529d6b927.png)

**Comment by tedhabermann** [comment]
Thanks very much Heidi for pointing this out. This file is an xsl created by Interactive Instruments for displaying gml codelist dictionaries in HTML. It is not a part of the ISO standard, although it is useful... I will check into whether I can include it in the standard repository...

**Comment by tedhabermann** [comment]
I received permission from the author to include this file in the repository and will add it with the next update or sooner... I will also attach a copy of the file to this issue.

**Comment by cportele** [comment]
Yes, please feel free to reuse the file (use as-is or make changes). It is a very simple transformation to make the code list somewhat easier to read in a browser than the raw XML.
